src/components/UserList/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from "react";
+import { useSearchParams } from "react-router-dom";
+import { formatLongDate } from "../../utils/date";
+import { DataTable } from "../DataTable";
+
+type UserListProps = {
+  data: any[];
+  isLoading?: boolean;
+};
+
+const DateCell = ({ cell: { value } }: { cell: { value: Date } }) =>
+  formatLongDate(value);
```

Comment:
```suggestion
  formatDate(value);
```

Reply:
We don't have an exported function formatDate. what date format are you expecting here.

Reply:
A date consistent with what we are using elsewhere - The long format shouldn't be used anywhere

Reply:
sure